dwave_networkx/drawing/qubit_layout.py

```diff
@@ -0,0 +1,237 @@
+    if linear_biases or quadratic_biases:
+        fig = plt.figure(1)
+        # cax = fig.add_axes([])
+        cax = fig.add_axes([.9, 0.2, 0.04, 0.6])  # left, bottom, width, height
+        mpl.colorbar.ColorbarBase(cax, cmap=cmap,
+                                  norm=mpl.colors.Normalize(vmin=-1 * vmag, vmax=vmag, clip=False),
+                                  orientation='vertical')
+
+
+def draw_embedding(G, layout, emb, embedded_graph=None, interaction_edges=None,
+                   chain_color=None, unused_color=None, cmap=None,
+                   **kwargs):
+    """Draws an embedding onto the graph G, according to layout.
+
+    If interaction_edges is not None, then only display the couplers in that
+    list.  If embedded_graph is not None, the only display the couplers between
+    chains with intended couplings according to embedded_graph.
+
+    Parameters
+    ----------
+    G : NetworkX graph
+        The graph to be drawn
+
+    layout : dict
+        A dict of coordinates associated with each node in G.  Should
+        be of the form {node: coordinate, ...}.  Coordinates will be
+        treated as vectors, and should all have the same length.
+
+    emb : dict
+        A dict of chains associated with each node in G.  Should be
+        of the form {node: chain, ...}.  Chains should be iterables
+        of qubit labels (qubits are nodes in G).
+
+    chain_color : dict (optional, default None)
+        A dict of colors associated with each key in emb.  Should be
+        of the form {node: rgba_color, ...}.  Colors should be length-4
+        tuples of floats between 0 and 1 inclusive.
+
+    chain_color : dict (optional, default None)
+        The color to use for nodes and edges of G which are not involved
+        in chains, and edges which are neither chain edges nor interactions.
+
+    embedded_graph : NetworkX graph (optional, default None)
+        A graph which contains all keys of emb as nodes.  If specified,
+        edges of G will be considered interactions if and only if they
+        exist between two chains of emb if their keys are connected by 
+        an edge in embedded_graph
+
+    interaction_edges : list (optional, default None)
+        A list of edges which will be used as interactions.
+
+    kwargs : optional keywords
+       See networkx.draw_networkx() for a description of optional keywords,
+       with the exception of the `pos` parameter which is not used by this
+       function. If `linear_biases` or `quadratic_biases` are provided,
+       any provided `node_color` or `edge_color` arguments are ignored.
+    """
+    try:
+        import matplotlib.pyplot as plt
+        import matplotlib as mpl
+    except ImportError:
+        raise ImportError("Matplotlib and numpy required for draw_chimera()")
+
+    if chain_color is None:
+        import matplotlib.cm
+        color = matplotlib.cm.get_cmap(kwargs.get("cmap"))
```

Comment:
looks like kwargs will never contain 'cmap', I think you meant to remove the 'cmap' kwarg from the function